Source/relay/TourGuide/Items of the Month/2023/Book of Facts.ash

```diff
@@ -0,0 +1,56 @@
+//Book of FActs
```

Comment:
thanks for the tile! as you noted on the server there are a few things remaining for when we can easily access mt_rand within tourguide but for now this is more than sufficient. 🥳